src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.view.html

```diff
@@ -1,113 +1,118 @@
-                        </div>
-                        <div class="col-xs-9">
-                            <ul class="mondial-opening">
-                                <li class="row"
-                                    data-ng-repeat="day in relay.opening track by day.day">
-                                    <div class="col-lg-2 col-md-4 col-xs-3"
-                                         data-ng-bind="('components_mondial_relay_' + day.day) | translate">
-                                    </div>
-                                    <div class="col-md-4 col-xs-4"
-                                         data-ng-repeat="hour in day.opening"
-                                         data-ng-bind="hour">
-                                    </div>
-                                </li>
-                            </ul>
+    <div class="row">
+        <div class="col-sm-6">
+            <leaflet id="{{$ctrl.mapId}}"
+                     data-markers="$ctrl.map.markers"
+                     data-center="$ctrl.map.center"
+                     data-event-broadcast="$ctrl.map.events"
+                     data-bounds="$ctrl.map.bounds">
+            </leaflet>
+        </div>
+        <div class="col-sm-6">
+            <fieldset class="mondial-results mt-5">
+                <legend class="ng-hide"
+                        aria-hidden="true"
+                        data-translate="components_mondial_relay_choose"></legend>
+                <ul class="list-group">
+                    <li class="list-group-item"
+                        data-ng-repeat="relay in $ctrl.foundRelays track by relay.id"
+                        data-ng-mouseover="$ctrl.markerHover($index)"
+                        data-ng-click="$ctrl.select(relay)"
+                        data-ng-class="{ 'active': relay.selected, 'marker-hover': relay.markerHover }">
+                        <div class="relay-cell">
+                            <div class="oui-radio">
+                                <input class="oui-radio__input"
+                                       name="oui-radio"
+                                       value="oui-radio-{{$index}}"
+                                       id="oui-radio-{{$index}}"
+                                       type="radio">
+                                <label class="oui-radio__label-container"
+                                       for="oui-radio-{{$index}}">
+                                    <span class="oui-radio__label">
+                                        <span class="mondial-marker"
+                                              data-ng-bind="relay.marker">
+                                        </span>
+                                        <strong class="mondial-relay-name"
+                                                data-ng-bind="relay.name">
+                                        </strong>
+                                        <span class="mondial-relay-caret glyphicon"
```

Comment:
`aria-hidden="true"` and alternative  for the screen-readers

Reply:
`aria-label` is enough for A11Y?

Reply:
The standard way of handling icons with a semantic meaning:
* add `aria-hidden` to the icon
* have another `span` with the text and add to this span the `sr-only` class

The advantage is that the `span.sr-only` will probably be displayed to users who disable CSS for their own